docs/src/pages/Composition.mdx

```diff
@@ -0,0 +1,45 @@
+export const meta = {
+  title: 'Composition',
+  previous: { name: 'Introduction', url: '/' },
+  next: { name: 'Shorthand Props', url: '/shorthand-props' },
+}
+
+## `as` prop
+
+Stardust UI provides a way to compose React components through the `as` prop. It allows to compose component features and props without adding extra nested components.
+
+```jsx
+<>
+  {/* 🧱 Each Stardust UI component has a default value for `as` prop */}
+  {/* Will output: <h1 class="ui-header" /> */}
+  <Header />
+  {/* Uses another tag: <h2 class="ui-header" /> */}
+  <Header as="h2" />
+</>
+```
```

Comment:
can you please add following:
```
Note: Using 'as' prop can change the behavior of the component, therefore Stardust does not handle accessibility and styling of the component if this prop is uses. It is the responsibility of the consumer to verify these aspects and handle them correctly.
```

Not entirely sure about the styling part, you are in better position to say that, but I think even for styling, you cannot guarantee that the styles will be applied properly if you use as.

Reply:
![image](https://user-images.githubusercontent.com/14183168/60337307-012dab80-99a3-11e9-9193-e5938dba2022.png)